### PR TITLE
build: Add windows-aarch64 to python binaries

### PR DESCRIPTION
**Pull request by ritchie46**

### CODE REVIEW

File: .github/workflows/release-python.yml [hunk 1]
```diff
@@ -92,11 +92,13 @@ jobs:
       fail-fast: false
       matrix:
         package: [polars, polars-lts-cpu, polars-u64-idx]
-        os: [ubuntu-latest, macos-13, windows-32gb-ram]
+        os: [ubuntu-latest, macos-13, windows-latest, windows-arm64-16gb]
         architecture: [x86-64, aarch64]
         exclude:
-          - os: windows-32gb-ram
+          - os: windows-latest
```
Comment (ritchie46) [inline .github/workflows/release-python.yml:98]: Public windows runners now have 16GB RAM. Seems enough.

File: .github/workflows/release-python.yml [hunk 2]
```diff
@@ -92,11 +92,13 @@ jobs:
...
             architecture: aarch64
+          - os: windows-arm64-16gb
+            architecture: x86-64
```
[the remark this answers is not in the document]
Comment (ritchie46) [reply .github/workflows/release-python.yml:101]: Don't we need to include all variants then?

**Comment by ritchie46** [comment]
I will take it for a spin. :sunglasses:

File: .github/workflows/release-python.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (stinodego) [reply .github/workflows/release-python.yml:101]: I'm pretty sure it's correct. The docs are here:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations

[the remark this answers is not in the document]
Comment (ritchie46) [reply .github/workflows/release-python.yml:101]: Right, I think you're right. I also do need to think harder with the include set. :laughing:

**Pull request merged**